.gitignore

```diff
@@ -19,3 +19,4 @@ spec/reports
 test/tmp
 test/version_tmp
 tmp
+.idea
```

Comment:
=D I still need to get my new laptop up to speed with my dotfiles.

But TBH I just commited this by mistake. xD I'll revert later.